godtools/App/Features/Onboarding/OnboardingQuickStartView.swift

```diff
@@ -0,0 +1,149 @@
+//
+//  OnboardingQuickStartView.swift
+//  godtools
+//
+//  Created by Robert Eldredge on 11/8/21.
+//  Copyright © 2021 Cru. All rights reserved.
+//
+
+import UIKit
+
+class OnboardingQuickStartView: UIViewController {
+    
+    enum DismissOnboardingTutorialType {
+        case readArticles
+        case tryLessons
+        case chooseTool
+    }
+    
+    private let viewModel: OnboardingQuickStartViewModelType
+    
+    private var skipButton: UIBarButtonItem?
+        
+    @IBOutlet weak private var titleLabel: UILabel!
+    @IBOutlet weak private var endTutorialButton: OnboardPrimaryButton!
+    @IBOutlet weak private var quickStartTableView: UITableView!
+    
+    required init(viewModel: OnboardingQuickStartViewModelType) {
+        
+        self.viewModel = viewModel
+
+        super.init(nibName: String(describing: OnboardingQuickStartView.self), bundle: nil)
+    }
+    
+    required init?(coder: NSCoder) {
+        
+        fatalError("init(coder:) has not been implemented")
+    }
+    
+    deinit {
+        
+        print("x deinit: \(type(of: self))")
+    }
+    
+    override func viewDidLoad() {
+        
+        super.viewDidLoad()
+        print("view didload: \(type(of: self))")
+        
+        setupLayout()
+        setupBinding()
+        
+        endTutorialButton.addTarget(self, action: #selector(handleEndTutorial), for: .touchUpInside)
+        
+        quickStartTableView.delegate = self
+        quickStartTableView.dataSource = self
+    }
+    
+    private func setupLayout() {
+        
+        titleLabel.text = viewModel.title
+        
+        endTutorialButton.setTitle(viewModel.endTutorialButtonTitle, for: .normal)
```

Comment:
Hey @reldredge71 can you move any configuring / observing of the viewModel into setupBinding() just to keep consistent with other views.